src/main/java/uk/gov/register/views/RecordView.java

```diff
@@ -35,27 +40,37 @@ public ObjectNode getRecordJson() {
         ObjectMapper objectMapper = Jackson.newObjectMapper();
         ObjectNode jsonNodes = objectMapper.convertValue(entry, ObjectNode.class);
         jsonNodes.remove("key");
-        jsonNodes.setAll(objectMapper.convertValue(itemView, ObjectNode.class));
+        ArrayNode items = jsonNodes.putArray("items");
+        itemViews.forEach( iv -> items.add(objectMapper.convertValue(iv, ObjectNode.class)));
         return jsonNodes;
     }
 
-    public Map<String, FieldValue> getContent() {
-        return itemView.getContent();
+    ArrayNode getFlatRecordJson() {
```

Comment:
We're introducing some inconsistency here. If you load an entry with two items, here is what you get for the TSV format when you download the record and entry, respectively:

`/record/402160.tsv`:
```entry-number	entry-timestamp	item-hashes	school	name	start-date	end-date	gender	religious-character	minimum-age	maximum-age	headteacher	website	address
3	2016-11-24T17:20:33Z	sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1	402161	First Steps	2007-11-06								10034660829
3	2016-11-24T17:20:33Z	sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04	402160	Options Education Programme	2007-11-06	2010-08-31							100101037877```

`/entry/3.tsv`:
```entry-number	entry-timestamp	item-hashes	key
3	2016-11-24T17:20:33Z	sha-256:088bb2124883bf2b82812f6d290cf860d9699fb17368e66ee442c3875162bfa1;sha-256:cbc12a2ff953c5751b34706990ff32af935942b4901254079457e6e5b5c35c04	402160
```

As you notice, we have the same column `item-hashes` in both outputs, yet in `record` we display each item on the entry as a separate row on the same entry number (effectively rendering  the new `item-hashes` column no more than it's predecessor `item-hash`) and in `entry` we put both hashes into `item-hashes` separated by a `;`.

We should standardise this and choose one approach.